src/styles/genStyl.js

```diff
@@ -0,0 +1,50 @@
+import vals from './vals.js';
+
+const genStyl = {
+  bubbleBox:  {
+    marginTop: '0.2rem',
+    marginBottom: '0.2rem',
+    opacity: '1',
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    maxWidth: '80%',
+    minHeight: '30px',
+    padding: '0',
+    borderRadius: '0.3rem',
+    backgroundColor: vals.bubbleBg,
+    boxShadow: '0.01rem 0.05rem 0 0 '+ vals.bubbleShadow
+  },
+  bubbleText: {
+    flex: '1 1 auto',
+    alignSelf: 'stretch',
+    display: 'inline-block',
+    minWidth: '5%',
+    margin: '0',
+    padding: '0.6rem 0.5rem',
+    display: 'inline-block'
+  },
+
+  botbubbleFlex: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'flex-start'
+  },
+
+  bubbleMarginPadding: {
+    paddingTop: '2rem',
+    marginLeft: '0.4rem',
+    marginRight: '0.4rem'
+  },
```

Comment:
These styles look very specific and not generic.

In general, prefer "good" components that can be used more generally, like a `<Bubble />` or something that have these styles